### ISSUE TITLE
Do not use nested scopes in examples

### ISSUE THREAD

**Issue by chrismccord** [phoenixframework/phoenix_guides#439]
I push folks away from nesting scopes because it gets confusing and is less clear. We should update the routing code to change this block:

```elixir
  scope "/", HelloPhoenix do
    pipe_through :browser

    resources "posts", PostController

    scope "/reviews" do
      pipe_through :review_checks

      resources "reviews", ReviewController
    end
  end
```

To this:

```elixir
  scope "/", HelloPhoenix do
    pipe_through :browser

    resources "posts", PostController
  end

  scope "/reviews", HelloPhoenix do
    pipe_through [:browser, :review_checks]

    resources "reviews", ReviewController
  end
```

Prose will also need tweaked talking about nested scopes.

**Comment by willykaram** [comment]
@chrismccord @lancehalvorsen I've made a PR to address the above issue. Pls LMK if you have any feedback, and I'll revise accordingly.

**Comment by willykaram** [comment]
As mentioned in the PR, apologies I made a sloppy error while squashing and then another error in my prior branch, and given the time of day it was just easier to create a new PR than to clean-up the error.

**Comment by lancehalvorsen** [comment]
closed by #455
